package-lock.json

```diff
@@ -6665,8 +6670,7 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true,
-          "optional": true
```

Comment:
Out of curiosity: What is the effect of removing the optional predicate here and in the other functions?

Reply:
I am not sure. This is just changed by npm automatically and I just commit it. I googled if I should commit it, but apparently I should.